core/src/test/java/io/substrait/type/parser/TestTypeParser.java

```diff
@@ -114,7 +114,7 @@ private void compoundTests(ParseToPojo.Visitor v) {
   private <T> void parameterizedTests(ParseToPojo.Visitor v) {
     test(v, pn.listE(pr.parameter("K")), "List?<K>");
     test(v, pr.structE(r.I8, r.I16, n.I8, pr.parameter("K")), "STRUCT<i8, i16, i8?, K>");
-    test(v, pn.parameter("any"), "any");
+    test(v, pn.parameter("any"), "any?");
```

Comment:
minor: I think it would be good to test both nullable and not nullable any
```java
    test(v, pr.parameter("any"), "any");
    test(v, pn.parameter("any"), "any?");
```